db/commitlog/commitlog.cc

```diff
@@ -94,7 +94,7 @@ class crc32_nbo {
 class db::cf_holder {
 public:
     virtual ~cf_holder() {};
-    virtual void release_cf_count(const cf_id_type&) = 0;
+    virtual void release_cf_count(const cf_id_type&, const replay_position&) = 0;
```

Comment:
nit: typo in commit log message: s/typo/type/

Reply:
fixed